### PR TITLE
Add the contact section heading background text

### LINKED ISSUE #164: Contact section main heading background shadow is missing
- Currently, the contact section main heading background shadow is missing
- Add this code,
````html
 <!-- Contact Section --> 
 <section id="contact"> 
   <h1>CONTACT ME</h1>
   <h3>CONTACT <span class="text-info">ME</span></h3> 
````
- To here in the contact section.
https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE/blob/a71b7d512553edaed7a6a84530445322f9a81f09/dark/index.html#L211-L213

### PR DESCRIPTION
**Pull request by maddy020**
## Purpose

The purpose of this Pull Request is to fix #164 <issue-number>

## Goals
Only some code to show the background need to to be update

## Approach
A new tag (h1) is required and need to insert in the section tag 

### Before

before
### After
after

### Preview Link

https://pr-172-NEW-YOSHITHA-WEBSITE.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: dark/index.html [hunk 1]
```diff
@@ -150,7 +150,7 @@ <h1>
       </div>
       <div id="tooltip">
         <span id="tooltiptext">Telegram</span>
-        <a href="https://t.me/YoshithaRathnayake2008" target="_blank "><img
+        <a href="https://img.icons8.com/fluency/48/telegram-app.png" target="_blank "><img
```
Comment (YoshithaRathnayake) [inline dark/index.html:153]: Remove this early changes @maddy020 
```suggestion
        <a href="https://t.me/YoshithaRathnayake2008" target="_blank "><img
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by maddy020** [comment]
I have changed the file 
Please review

File: dark/index.html [hunk 2]
```diff
@@ -151,7 +151,7 @@ <h1>
       <div id="tooltip">
         <span id="tooltiptext">Telegram</span>
         <a href="https://t.me/YoshithaRathnayake2008" target="_blank "><img
-            src="https://img.icons8.com/color/48/000000/telegram-app--v1.png" alt="Telegram Icon" /></a>
+            src="https://img.icons8.com/fluency/48/telegram-app.png" alt="Telegram Icon" /></a>
```
Comment (YoshithaRathnayake) [inline dark/index.html:154]: Change this link @maddy020 

```suggestion
            src="https://img.icons8.com/color/48/000000/telegram-app--v1.png" alt="Telegram Icon" /></a>
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html (hunk 2 above)
Comment (maddy020) [reply dark/index.html]: Has changed 
Please review

File: dark/index.html [hunk 3]
```diff
@@ -151,7 +151,7 @@ <h1>
       <div id="tooltip">
         <span id="tooltiptext">Telegram</span>
         <a href="https://t.me/YoshithaRathnayake2008" target="_blank "><img
-            src="https://img.icons8.com/color/48/000000/telegram-app--v1.png" alt="Telegram Icon" /></a>
+            src="https://img.icons8.com/color/48/000000/telegram-app--v1" alt="Telegram Icon" /></a>
```
Comment (YoshithaRathnayake) [inline dark/index.html:154]: `.png` is missing in the link @maddy020 
Add it.
```suggestion
            src="https://img.icons8.com/color/48/000000/telegram-app--v1.png" alt="Telegram Icon" /></a>
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — APPROVED** [review]
Thank You! @maddy020

**Comment by maddy020** [comment]
> Thank You! @maddy020

Your welcome Bro !

**Pull request merged**